cypress/e2e/inventory/holdings/create-new-marc-holdings.cy.js

```diff
@@ -156,3 +158,87 @@ describe('Create holding records with MARC source', () => {
     },
   );
 });
+
+describe('MARC › MARC Holdings', () => {
+  const servicePoint = ServicePoints.getDefaultServicePointWithPickUpLocation();
+  const testData = {
+    tag851: '851',
+    tag852: '852',
+    tag851value: '$a subfield with any value',
+  };
+  const marcFile = {
+    marc: 'oneMarcBib.mrc',
+    fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+    jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+  };
+
+  let defaultLocation;
+  let user;
+
+  before(() => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.uiQuickMarcQuickMarcHoldingsEditorCreate.gui,
+      Permissions.uiQuickMarcQuickMarcHoldingsEditorAll.gui,
+    ]).then((createdUserProperties) => {
+      ServicePoints.createViaApi(servicePoint);
+      defaultLocation = Location.getDefaultLocation(servicePoint.id);
+      testData.tag852_B_value = `$b ${defaultLocation.code}`;
+      testData.tag852_B_E_values = `$b ${defaultLocation.code} $e Test`;
+      Location.createViaApi(defaultLocation);
+      user = createdUserProperties;
+      cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(
```

Comment:
Can this be done via API calls?

Reply:
I'll check

Reply:
Haven't find a way how to import marc file through API